uniro_admin_frontend/src/components/log/logList.tsx

```
@@ -0,0 +1,15 @@
+import React from "react";
+import LogCard from "./logCard";
+import { logCardData } from "../../data/mock/logMockData";
+
+const LogList = () => {
+  return (
+    <div className="flex-1 flex flex-col w-full h-full overflow-y-scroll px-1 space-y-2 mb-2 ">
+      {logCardData.map((log) => {
+        return <LogCard date={log.date} change={log.change} />;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)